From 83b2c2f1aac5f4c73d8019d6cd4a9f3aeef1fccc Mon Sep 17 00:00:00 2001
From: Alexandru Costache <alexandru@balena.io>
Date: Tue, 2 Jun 2020 15:24:54 +0200
Subject: [PATCH] Silent console in production

GD_FLG_SILENT flag may get set or cleared
from other places inside uboot. This patch
disables console output alltogether.

Upstream-status: Inappropriate [configuration]
Signed-off-by: Alexandru Costache <alexandru@balena.io>
---
 common/console.c | 1 +
 1 file changed, 1 insertion(+)

diff --git a/common/console.c b/common/console.c
index e398530a13..870869789a 100644
--- a/common/console.c
+++ b/common/console.c
@@ -548,6 +548,7 @@ void putc(const char c)
 
 void puts(const char *s)
 {
+    return;
 #ifdef CONFIG_SANDBOX
 	/* sandbox can send characters to stdout before it has a console */
 	if (!gd || !(gd->flags & GD_FLG_SERIAL_READY)) {
-- 
2.17.1

